ihmc-communication/src/main/java/us/ihmc/communication/PerceptionAPI.java

```diff
@@ -251,7 +251,8 @@ public final class PerceptionAPI
    public static final ROS2Topic<ImageMessage> HEIGHT_MAP_LOCAL = PERCEPTION_MODULE.withOutput().withTypeName(ImageMessage.class).withSuffix("height_map_local");
 
    public static final ROS2Topic<Empty> REQUEST_LIDAR_SCAN = PERCEPTION_MODULE.withSuffix("request_lidar_scan").withType(Empty.class);
-   public static final ROS2Topic<Empty> REQUEST_HEIGHT_MAP = PERCEPTION_MODULE.withSuffix("request_height_map").withType(Empty.class);
+   public static final ROS2Topic<Empty> REQUEST_OUSTER_HEIGHT_MAP = PERCEPTION_MODULE.withSuffix("request_ouster_height_map").withType(Empty.class);
```

Comment:
We only use one height map, can we leave this how it was? This adds complexity when its not needed right now